cypress/e2e/Labels/tests/singleDynamicGroupedLabelCRUDTests.ts

```diff
@@ -19,14 +19,27 @@ import {
 
 export const SingleDynamicGroupedLabelCRUDTests = () =>
   describe("Single Dynamic Grouped Label Tests", () => {
-    describe("Able to Create / Read / Update / Delete", () => {
+    const groupNameC = generateUniqueName("initialDynamicGroupedCRUD");
+
+    it("Create dynamic grouped label group", () => {
+      createNewLabel({
+        groupName: groupNameC,
+        condition: DYNAMIC_GROUPED_LABEL_QUERY_TEXT_1,
+        type: LABEL_TYPES.DYNAMIC_GROUPED,
+      });
+    });
+
+    describe("Able to Read / Update / Delete", () => {
       const groupName = generateUniqueName("initialDynamicGroupedCRUD");
 
-      it("Create dynamic grouped label group", () => {
-        createNewLabel({
-          groupName: groupName,
-          condition: DYNAMIC_GROUPED_LABEL_QUERY_TEXT_1,
-          type: LABEL_TYPES.DYNAMIC_GROUPED,
+      beforeEach(() => {
```

Comment:
Just curious: is there a way in cypress to exit the test if createLabel fails? (it would be nice to exit and not continue with the rest of checks)

Reply:
I'll look into it 👍

Reply:
it throws an error, which ends the test. So this is skipping except with more information.